h5c/vic/src/vic-webapp/src/app/container-view/container-view.template.html

```diff
@@ -6,18 +6,29 @@
           <clr-alert-item *ngIf="errorObj.type === 'ssl_cert'">
             <span class="alert-text">
               <!-- TODO: i18n -->
-              Failed to verify the VIC machine server at endpoint https://{{ errorObj.payload }}:8443. Your browser may be blocking the connection because of the untrusted SSL certificate.
+              Failed to verify the VIC machine server at endpoint https://{{ errorObj.payload }}:8443. Please check the following:<br/>
```

Comment:
I think we need to verify all of this text with Stuart and/or UX

Reply:
@stuclem @lweitzman could you please share your thoughts on this please?

Reply:
How about "Failed to verify the `vic-machine` server at endpoint https://{{ errorObj.payload }}:8443. Please check the following:"

We shouldn't use "VIC" as an abbreviation, but since `vic-machine` is a component of VIC, that's fine.